<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoinsWindow</class>
 <widget class="QMainWindow" name="CoinsWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>920</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tiempo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="coinFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1931</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1901</width>
       <height>891</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="coinLayout"/>
    </widget>
    <widget class="QPushButton" name="goBack">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>950</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Volver al tablero</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
